client.go

```diff
@@ -63,27 +65,27 @@ func checkConfig(c *cli.Context) error {
 }
 
 // checkList sends a message to the list and waits for the reply
-func checkList(list *sda.EntityList, wg *sync.WaitGroup) {
+func checkList(list *sda.EntityList, descs []string, wg *sync.WaitGroup) {
 	defer wg.Done()
 	serverStr := ""
-	for _, s := range list.List {
-		serverStr += s.Addresses[0] + " "
+	for i, s := range list.List {
+		name := strings.Split(descs[i], " ")[0]
+		serverStr += fmt.Sprintf("%s_%s ", s.Addresses[0], name)
 	}
 	dbg.Lvl3("Sending message to: " + serverStr)
 	msg := "verification"
+	fmt.Print("[+] Checking server(s) ", serverStr, ": ")
 	sig, err := signStatement(strings.NewReader(msg), list)
 	if err != nil {
 		fmt.Fprintln(os.Stderr,
-			fmt.Sprintf("[-] Error '%v' while contacting servers: %s",
-				err, serverStr))
+			fmt.Sprintf("Error '%v'", err))
 	} else {
 		err := verifySignatureHash([]byte(msg), sig, list)
 		if err != nil {
 			fmt.Println(os.Stderr,
-				fmt.Sprintf("[-] Received signature from %s was invalid: %v",
-					serverStr, err))
+				fmt.Sprintf("Invalid signature: %v", err))
```

Comment:
No we won't see which server failed anymore? I guess that's OK because we won't use checkList in go-rountines again? (same holds for some lines above in the error case)